### PR TITLE
EAPI5 cleanup

### PR DESCRIPTION
**Pull request by jsmolic**

### CODE REVIEW

File: sys-auth/polkit-pkla-compat/polkit-pkla-compat-0.1-r1.ebuild [hunk 1]
```diff
@@ -10,22 +10,20 @@ SRC_URI="https://releases.pagure.org/${PN}/${P}.tar.xz"
...
 	app-text/docbook-xsl-stylesheets
 	dev-libs/libxslt
 	virtual/pkgconfig"
 
-DOCS="AUTHORS NEWS README"
-
 src_install() {
 	default
-	fowners -R root:polkitd /etc/polkit-1/localauthority
+	fowners -R root:polkitd /etc/polkit-1/localauthority || die
```
Comment (SoapGentoo) [inline sys-auth/polkit-pkla-compat/polkit-pkla-compat-0.1-r1.ebuild:24]: `fowners` is a gentoo built-in, not an external command

File: sys-block/noflushd/noflushd-2.8-r1.ebuild [hunk 2]
```diff
@@ -10,16 +10,15 @@ SRC_URI="mirror://sourceforge/noflushd/${PN}-${PV}.tar.gz"
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="amd64 ppc x86"
-IUSE=""
 
 src_configure() {
 	econf \
-		--with-docdir=/usr/share/doc/${PF} \
+		--with-docdir="${EPREFIX}"/usr/share/doc/${PF} \
 		--with-initdir=/etc/init.d
```
Comment (SoapGentoo) [inline sys-block/noflushd/noflushd-2.8-r1.ebuild:17]: this one too?

File: sys-block/noflushd/noflushd-2.8-r1.ebuild [hunk 3]
```diff
@@ -10,16 +10,15 @@ SRC_URI="mirror://sourceforge/noflushd/${PN}-${PV}.tar.gz"
...
 }
 
 src_install() {
-	emake install DESTDIR="${D}"
+	emake DESTDIR="${D}" install
 	dodoc NEWS
```
Comment (SoapGentoo) [inline sys-block/noflushd/noflushd-2.8-r1.ebuild:22]: `default`?

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: sys-block/noflushd/noflushd-2.8-r1.ebuild (hunk 3 above)
Comment (jsmolic) [reply sys-block/noflushd/noflushd-2.8-r1.ebuild]: Yep, done :)

File: sys-auth/polkit-pkla-compat/polkit-pkla-compat-0.1-r1.ebuild (hunk 1 above)
Comment (jsmolic) [reply sys-auth/polkit-pkla-compat/polkit-pkla-compat-0.1-r1.ebuild]: Oh that's right, thanks :+1:

File: sys-block/noflushd/noflushd-2.8-r1.ebuild (hunk 2 above)
Comment (jsmolic) [reply sys-block/noflushd/noflushd-2.8-r1.ebuild]: Yes

**Review by SoapGentoo — APPROVED** [review]
didn't we sign off on these already?

**Comment by jsmolic** [comment]
@SoapGentoo Thanks, couldn't remember so wanted to check to make sure ;)

**Pull request closed without merging**